src/common/opencl.c

```diff
@@ -475,10 +467,23 @@ static int dt_opencl_device_init(dt_opencl_t *cl, const int dev, cl_device_id *d
   cl->crc = crc32(cl->crc, (const unsigned char *)infostr, strlen(infostr));
 
   const gboolean newdevice = dt_opencl_read_device_config(dev);
+  const gboolean blacklist = dt_opencl_check_driver_blacklist(deviceversion);
```

Comment:
for clarity I would name this variable `is_blacklisted`.

Reply:
sure